go.mod

```diff
@@ -4,79 +4,42 @@ go 1.13
 
 require (
 	github.com/BurntSushi/toml v0.3.1
-	github.com/MakeNowJust/heredoc v0.0.0-20171113091838-e9091a26100e // indirect
 	github.com/Masterminds/semver/v3 v3.0.3
 	github.com/Masterminds/sprig/v3 v3.0.2
-	github.com/Masterminds/vcs v1.13.0
-	github.com/Microsoft/go-winio v0.4.12 // indirect
-	github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a
-	github.com/containerd/containerd v1.3.0
+	github.com/Masterminds/vcs v1.13.1
+	github.com/asaskevich/govalidator v0.0.0-20200108200545-475eaeb16496
+	github.com/containerd/containerd v1.3.2
 	github.com/cyphar/filepath-securejoin v0.2.2
-	github.com/deislabs/oras v0.7.0
+	github.com/deislabs/oras v0.8.1-0.20200203195554-169a5428f68f
```

Comment:
I see that `v0.8.1` is out the door. It is the latest commit on the ORAS repo, too. This references using a different commit in `169a5428f68f`. Is there a reason to not use `v0.8.1` itself?

Reply:
whoops, just fixed